web/admin/pages/users/[did].vue

```diff
@@ -17,18 +21,31 @@ async function loadProfile() {
 const auditEvents: Ref<AuditEvent[]> = ref([]);
 
 async function loadEvents() {
+  error.value = null;
+
   const response = await api.listAuditEvents({
     filterSubjectDid: subject.value.did,
+  }).catch((err) => {
+    error.value = { rawMessage: err.rawMessage };
+    return {
+      auditEvents: [],
+    };
   });
   auditEvents.value = response.auditEvents;
 }
 
 async function comment(comment: string) {
+  error.value = null;
+
   await api.createCommentAuditEvent({
     subjectDid: subject.value.did,
     comment,
+  }).catch((err) => {
+    error.value = { rawMessage: err.rawMessage };
   });
-  await loadEvents();
+
+  if (!error.value)
+    await loadEvents();
```

Comment:
**thought**: i don’t mind the audit event log to refresh even if the comment failed. WDYT?

Reply:
It can cause two error message to happened at the same time, so i think event log should not be fetched, but i am unsure